CONTRIBUTING.md

```diff
@@ -127,5 +127,5 @@ testing.
  [ar:gitter]: https://gitter.im/syslog-ng/syslog-ng
  [ar:mailing-list]: http://lists.balabit.com/mailman/listinfo/syslog-ng
  [ar:issue-tracker]: https://github.com/syslog-ng/syslog-ng/issues
- [ar:issues:help]: https://github.com/syslog-ng/syslog-ng/labels/help
```

Comment:
Please also update the text `help`, it is confusing this way.

Reply:
Uhh, thanks!